pkg/cvo/cvo.go

```diff
@@ -210,6 +212,7 @@ func New(
 	optr.proxyLister = proxyInformer.Lister()
 	optr.cmConfigLister = cmConfigInformer.Lister().ConfigMaps(internal.ConfigNamespace)
 	optr.cmConfigManagedLister = cmConfigManagedInformer.Lister().ConfigMaps(internal.ConfigManagedNamespace)
+	optr.cmConfigAdminGatesLister = cmConfigAdminGatesInformer.Lister().ConfigMaps(internal.ClusterVersionNamespace)
```

Comment:
admin gates are what we're using this for now, but the actual informer is covering all ConfigMaps in our namespace, of which there may be more later.  So maybe `optr.cmClusterVersion` or some such instead?

Reply:
or maybe we move `admin-gates` into `openshift-config-managed` and `admin-acks` into `openshift-config`?  One benefit for us is that we already have informers for those, but another benefit is that admins touching some, but not all, ConfigMaps in `openshift-config` sounds better to me than training folks to touch some, but not all, ConfigMaps in `openshift-cluster-version`.  Would probably need an enhancement bump, if we make this pivot.

Reply:
Renamed relevant names changing `AdminGates` -> `ClusterVersion`.